storage3/_async/file_api.py

```diff
@@ -29,12 +29,18 @@ async def _request(
     ) -> Response:
         headers = headers or {}
         response = await self._client.request(
-            method, url, headers={**self._headers, **headers}, json=json, files=files
+            method,
+            url,
+            headers={**self._client.headers, **headers},
```

Comment:
`self._client.headers` isn't needed here again, httpx will automatically be sending them with all requests

Reply:
Right, but if I just set a dict of headers, won't it overwrite the headers set in the client?

Reply:
Nope, in the docs, they say they merge client level headers and request level headers

Reply:
Right, so you don't need to merge them again here right?

Reply:
Nope, 5 min.